magentoerpconnect/magento_model.py

```diff
@@ -186,17 +187,23 @@ def check_magento_structure(self):
 
     @api.multi
     def synchronize_metadata(self):
-        session = ConnectorSession(self.env.cr, self.env.uid,
-                                   context=self.env.context)
-        for backend in self:
-            for model in ('magento.website',
-                          'magento.store',
-                          'magento.storeview'):
-                # import directly, do not delay because this
-                # is a fast operation, a direct return is fine
-                # and it is simpler to import them sequentially
-                import_batch(session, model, backend.id)
-        return True
+        try:
+            session = ConnectorSession(self.env.cr, self.env.uid,
+                                       context=self.env.context)
+            for backend in self:
+                for model in ('magento.website',
+                              'magento.store',
+                              'magento.storeview'):
+                    # import directly, do not delay because this
+                    # is a fast operation, a direct return is fine
+                    # and it is simpler to import them sequentially
+                    import_batch(session, model, backend.id)
+            return True
+        except Exception as e:
+            _logger.error(e.message, exc_info=True)
+            raise Warning(
+                _(u"Check your configuration, we can't get the data.\
+                Here is the error:\n%s") % str(e).decode('utf-8', 'ignore'))
```

Comment:
Could you write it in the following manner:

```python
raise Warning(
    _(u"Check your configuration, we can't get the data. "
      u"Here is the error:\n%s") % str(e).decode('utf-8', 'ignore'))
```

So we don't get blanks in the string?